idb/postgres/postgres_integration_test.go

```diff
@@ -668,4 +669,247 @@ func TestIgnoreDefaultFrozenConfigUpdate(t *testing.T) {
 	// default-frozen = true
 	assertAccountAsset(t, db, test.AccountA, assetid, false, total)
 	assertAccountAsset(t, db, test.AccountB, assetid, false, 0)
-}
\ No newline at end of file
+}
+
+func TestDestroyAssetBasic(t *testing.T) {
+	db, connStr, shutdownFunc := setupPostgres(t)
+	defer shutdownFunc()
+
+	pdb, err := idb.IndexerDbByName("postgres", connStr, nil, nil)
+	assert.NoError(t, err)
+
+	cache, err := pdb.GetDefaultFrozen()
+	assert.NoError(t, err)
+
+	assetID := uint64(3)
+
+	// Create an asset.
+	{
+		_, txnRow := test.MakeAssetConfigOrPanic(test.Round, 0, assetID, 4, 0, false, "uu", "aa", "",
+			test.AccountA)
+
+		state := getAccounting(test.Round, cache)
+		err := state.AddTransaction(txnRow)
+		assert.NoError(t, err)
+
+		err = pdb.CommitRoundAccounting(state.RoundUpdates, test.Round, &itypes.Block{})
+		assert.NoError(t, err, "failed to commit")
+	}
+	// Destroy an asset.
+	{
+		_, txnRow := test.MakeAssetDestroyTxn(test.Round + 1, assetID)
+
+		state := getAccounting(test.Round + 1, cache)
+		err := state.AddTransaction(txnRow)
+		assert.NoError(t, err)
+
+		err = pdb.CommitRoundAccounting(state.RoundUpdates, test.Round + 1, &itypes.Block{})
+		assert.NoError(t, err, "failed to commit")
+	}
+
+	// Check that the asset is deleted.
+	{
```

Comment:
You could extend/use `assertAccountAsset` or make a new similar function for these checks.

Reply:
done